src/silly_sprites/event/events.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "base_handler.hpp"
+#include <string>
+
+namespace sly::event {
+    struct Message final : public Base<Message> {
+    private:
+        std::string m_message;
+
+    public:
+        Message(std::string const& message) : m_message{ message } {};
```

Comment:
Constructor should be `explicit` and its parameter should be taken by value and then `std::move`d into the data member.